kano_registration_gui/RegistrationScreen.py

```diff
@@ -105,43 +105,43 @@ def _on_register_button(self, widget=None):  # TODO: refactor this
         # screen. However there is a small chance someone could take the
         # username while the user is in the process of registering
         if not success:
-            if text.strip() == "Cannot register, problem: " \
-               "Username already registered":
+            if text.strip() == _("Cannot register, problem: " \
```

Comment:
So.. this `text` is coming from `kano_world.functions`, which returns now also a translated string, but which gets formatted by whatever `request_wrapper` will return.. And so on.. Need to be careful when testing this one..